services/121-service/test/registrations/duplicates/get-duplicates.test.ts

```diff
@@ -0,0 +1,222 @@
+import { RegistrationStatusEnum } from '@121-service/src/registration/enum/registration-status.enum';
```

Comment:
I did not add any unit tests. The reason for this is 90 % of logic is in the queries. And therefore the scenario's I could think of are all covered in integration tests. Do you see opportunities for unit testing @diderikvw